xbmc/music/MusicDatabase.cpp

```diff
@@ -2037,11 +2047,14 @@ bool CMusicDatabase::GetArtist(int idArtist, CArtist& artist, bool fetchAll /* =
       return false;
     if (nullptr == m_pDS)
       return false;
+    if (nullptr == m_pDS2)
+      return false;
 
     if (idArtist == -1)
       return false; // not in the database
 
     std::string strSQL;
+    std::string debugSQL;
```

Comment:
Remove the declaration here and move it to line 2073, where you actually need this variable.

Reply:
Done.